examples/images.yaml

```diff
@@ -0,0 +1,14 @@
+# Dummy image tags
+# Image tags from the latest release
+cemanager:
+  image: gcr.io/coder/coder-service:2.0.0
```

Comment:
@deansheather Renamed release -> images and made dummy version tags. This doesn't matter for any of the linters currently in-use, but `kube-score` does check that we're not using the `default/latest` tag.